services/QuillLMS/spec/workers/save_activity_session_concept_results_worker_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe SaveActivitySessionConceptResultsWorker, type: :worker do
+
+  context '#perform' do
+    let(:old_concept_result) { create(:old_concept_result) }
+
+    it 'should save new ConceptResult records based on an array of OldConceptResult ids' do
+      expect { subject.perform(old_concept_result.id) }
+        .to change(ConceptResult, :count).by(1)
+      expect(old_concept_result.reload.concept_result).to be
```

Comment:
I bet it's one of those rspec wrinkles but the need for `reload` in line 13 but not in line 19 is odd.

Reply:
Actually, the inconsistency was just something that happened without planning.  Good call out since it turns out the `reload` is unnecessary.  Removed.